src/app/3d/qgs3dmaptoolpolygon.h

```diff
@@ -0,0 +1,56 @@
+/***************************************************************************
+    qgs3dmaptoolpolygon.h
+    ---------------------
+    begin                : February 2025
+    copyright            : (C) 2025 by Matej Bagar
+    email                : matej dot bagar at lutraconsulting dot co dot uk
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGS3DMAPTOOLPOLYGON_H
+#define QGS3DMAPTOOLPOLYGON_H
+
+#include "qgs3dmaptoolpointcloudchangeattribute.h"
+
+#include <QPointF>
+
+class QgsRubberBand3D;
+class QgsPointXY;
+
+class Qgs3DMapToolPolygon : public Qgs3DMapToolPointCloudChangeAttribute
```

Comment:
please rename the class to make it clear it is related to point cloud editing...